### PR TITLE
Cpptrace

### PR DESCRIPTION
**Pull request by Trial97**
Please do not review(here just to track my progress)

### CODE REVIEW

File: launcher/ui/pages/modplatform/ResourcePage.cpp [hunk 1]
```diff
@@ -254,10 +254,7 @@ void ResourcePage::updateSelectionButton()
 
     m_ui->resourceSelectionButton->setEnabled(true);
     if (auto current_pack = getCurrentPack(); current_pack) {
-        if (current_pack->versionsLoaded && current_pack->versions.empty()) {
-            m_ui->resourceSelectionButton->setEnabled(false);
-            qWarning() << tr("No version available for the selected pack");
-        } else if (!current_pack->isVersionSelected(m_selected_version_index))
+        if (!current_pack->isVersionSelected(m_selected_version_index))
```
Comment (TheKodeToad) [inline launcher/ui/pages/modplatform/ResourcePage.cpp:257]: I think you included this by mistake
Edit: Yes I know
image
image

Comment (Trial97) [reply launcher/ui/pages/modplatform/ResourcePage.cpp:257]: Se that is why I said no review.
No, I did not revert the crash fix by mistake.
I need a way to crash the application to test whether I get the stack trace.
But still, something I did wrong as on Windows it doesn't yet work :)

**Comment by Ryex** [comment]
So, on windows SIGSEGV is actually a "hardware exception"
https://stackoverflow.com/questions/41243855/windows-handle-segfaults-in-all-threads
looks like the missing ingredient is some compiler flags.
https://learn.microsoft.com/en-us/cpp/build/reference/eh-exception-handling-model?view=msvc-170
we want to exchanged `/EHsc` for `/EHa`

**Comment by Trial97** [comment]
That did not work: either windows is really special(and at this point, it may be) or I am missing something.
I tried every freaking thing I could think of and still got no results. 
Any other ideas?

**Comment by Trial97** [comment]
closing in favor of: https://github.com/Trial97/prism_gdb
Is too hard to capture the stacktrace for Windows inside the app, so I made a tutorial on how to wrap the app in gdb and give that to the users for crashes

**Pull request closed without merging**